cmd/info.go

```diff
@@ -40,6 +40,10 @@ func newInfoCmd() *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			jsonOutput, _ := cmd.Flags().GetBool("json")
 			token, _ := cmd.Flags().GetString("access-token")
+			if token == "" {
+				token = os.Getenv("REANA_ACCESS_TOKEN")
+			}
+
```

Comment:
Minor: do we need these empty lines after this check? IMO it logically separates a `token` from `serverURL` which are kind of related, WDYT?

Reply:
Sure, I just thought it was too bloated but maybe it makes sense to put them together, I can just that